JitsiConference.js

```diff
@@ -1558,17 +1709,29 @@ JitsiConference.prototype.isCallstatsEnabled = function() {
 /**
  * Handles track attached to container (Calls associateStreamWithVideoTag method
  * from statistics module)
- * @param track the track
+ * @param {JitsiLocalTrack|JitsiRemoteTrack} track the track
  * @param container the container
  */
 JitsiConference.prototype._onTrackAttach = function(track, container) {
-    const ssrc = track.getSSRC();
+    const isLocal = track.isLocal();
+    let ssrc = null;
+
+    if (isLocal) {
+        // Local tracks have SSRC stored on per peer connection basis
+        const peerConnection
```

Comment:
This logic there is about logging association between video tag and local SSRC to CallStats. But because we do CallStats only for the JVB it tries to get it from the JVB connection. I will add a comment.